packages/ai/src/ai/common/store.py

```
@@ -154,7 +154,7 @@ def _processFullTables(self, documents: Dict[Tuple[str, int], Doc]) -> Dict[Tupl
 
             # Update the score if this one is higher
             if doc.score > tableDocs[tableKey].score:
-                tableDocs[tableKey] = doc.score
+                tableDocs[tableKey].score = doc.score
```

Ruff (0.15.12)

[warning] Replace `if` statement with `max` call

Replace with `max` call

(PLR1730)